gloo/common/memory.h

```diff
@@ -140,7 +144,12 @@ class ShareableNonOwningPtr final {
     ptr_.reset();
     // Wait for all shared_ptr's to T have been released
     while (!weak.expired()) {
+#ifdef _WIN32
+      //Due to here sleep time is 0, does not want implement a usleep on windows
+      Sleep(0);
+#else
       usleep(0);
+#endif
     }
```

Comment:
`#include <thread>` above, then just use `std::this_thread::yield()` for a cross-platform version with the same effect.